algorithms/kernel/svm/oneapi/cl_kernels/svm_kernels.cl

```diff
@@ -0,0 +1,94 @@
+/* file: svm_kernels.cl */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of SVM kernels.
+//--
+*/
+
+#ifndef __SVM_KERNELS_CL__
+#define __SVM_KERNELS_CL__
+
+#include <string.h>
+
+#define DECLARE_SOURCE_DAAL(name, src) static const char *(name) = #src;
+
+DECLARE_SOURCE_DAAL(
+    clKernelSVM,
+
+    __kernel void makeInversion(const __global algorithmFPType * const x, __global algorithmFPType * res) {
+        const int i = get_global_id(0);
+        res[i]      = -x[i];
+    }
+
+    __kernel void makeRange(__global int * x) {
+        const int i = get_global_id(0);
+        x[i]        = i;
+    }
+
+    __kernel void checkUpper(const __global algorithmFPType * const y, const __global algorithmFPType * const alpha, const algorithmFPType C,
+                             __global int * indicator) {
+        const int i  = get_global_id(0);
+        indicator[i] = (y[i] > 0 && alpha[i] < C) || (y[i] < 0 && alpha[i] > 0);
+    }
```

Comment:
I would not want to change the names, since such are used in all articles. Concerning C. We even have some public parameter name. It will be strange to have big and small C.